src/ansys/fluent/core/workflow.py

```diff
@@ -499,10 +575,29 @@ def get_state(self, explicit_only=False) -> dict:
             Whether to only include explicitly set values,
             otherwise all values are included.
```

Comment:
```suggestion
            Whether to only include explicitly set values.
            Otherwise, all values are included.
```
While you could use a semicolon where you currently have the comma, tech doc generally avoids the use of semicolons in favor of short, concise sentences.